GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -129,16 +129,16 @@ private JsonTracer CreateTracer(GVFSEnlistment enlistment, EventLevel verbosity,
             string mountId = null;
 
             GitProcess git = new GitProcess(enlistment);
-            GitProcess.Result configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.EnlistmentId);
-            if (!configResult.HasErrors)
+            GitProcess.ConfigResult configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.EnlistmentId);
+            if (configResult.TryParseAsString(out enlistmentId, out string _))
             {
-                enlistmentId = configResult.Output.Trim();
+                enlistmentId = enlistmentId.Trim();

```

Comment:
Check if `enlistmentId` is `null` (or allow `TryParseAsString` to take a default value and pass in `string.Empty`)

Reply:
Now defaults to string.Empty